src/frontend/next/src/components/AdminButtons.tsx

```diff
@@ -0,0 +1,3 @@
+const AdminButtons = () => <h3> AdminButtons</h3>;
+
+export default AdminButtons;
```

Comment:
FYI: @rogercyyu and @sonechca we just need to co-ordinate this.  I think it's fine to land this now, but if the AdminButtons go in first (#1515), then let's rebase and remove this.